src/drivers/imx/interrupt.c

```diff
@@ -70,54 +70,24 @@
  * usage in this driver.
  */
 
-#if defined CONFIG_IMX8
-#define IRQSTR_BASE_ADDR	0x510A0000
-#endif
-
-#if defined CONFIG_IMX8X
-#define IRQSTR_BASE_ADDR	0x51080000
-#endif
-
-#if defined CONFIG_IMX8M
-#define IRQSTR_BASE_ADDR	0x30A80000
-#endif
-
 /* The MASK, SET (unused) and STATUS registers are 512-bit registers
  * split into 16 32-bit registers that we can directly access.
  *
  * To get the proper register for the shared interrupt irq, we do
  * IRQSTR_CH_MASK(IRQSTR_INT_REG(irq)) (MASK can be replaced by SET or
  * STATUS).
  *
- * The interrupts are mapped in the registers in the following way:
- * Interrupts 480-511 at offset 0
- * Interrupts 448-479 at offset 1
- * Interrupts 416-447 at offset 2
- * ...
- * Interrupts 64-95 at offset 13
- * Interrupts 32-63 at offset 14
- * Interrupts 0-31 at offset 15
+ * The interrupt mapping to registers is defined in platform specific
+ * headers.
```

Comment:
is defined in interrupt.h platform specific header? Is this more accurate?

Reply:
Is it better now?